app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -399,6 +399,11 @@ def vm_acquire_mks_ticket(vm, options = {})
     end
     alias_method :vm_remote_console_mks_acquire_ticket, :vm_acquire_mks_ticket
 
+    def vm_acquire_ticket(vm, options = {})
+      invoke_vim_ws(:acquireTicket, vm, options[:user_event], options[:ticket_type])
+    end
+    alias_method :vm_remote_console_acquire_ticket, :vm_acquire_ticket
```

Comment:
Yeah that works, and actually `AcquireMksTicket` is deprecated as of `4.1` so we can make that a wrapper around this with the `mks` ticket type